sfs/time/drivingfunction.py

```diff
@@ -255,3 +256,194 @@ def apply_delays(signal, delays):
     for column, row in enumerate(delays_samples):
         out[row:row + len(data), column] = data
     return util.DelayedSignal(out, samplerate, offset_samples / samplerate)
+
+
+def nfchoa_25d_plane(x0, r0, npw, max_order=None, c=None, fs=44100, normalize=True):
+    """Vritual plane wave by 2.5-dimensional NFC-HOA.
+
+    Parameters
+    ----------
+    x0 : (N, 3) array_like
+        Sequence of secondary source positions.
+    r0 : float
+        Radius of the circular secondary source distribution
+    npw : (3,) array_like
+        Unit vector (propagation direction) of plane wave.
+    max_order : int
+        Ambisonics order
+    c : float, optional
+        Speed of sound
```

Comment:
`fs` and `normalize` are not documented

Reply:
OK.

I will remove the `normalize` option since it does not make sense not normalizing the filter. The option was used to demonstrate the effect of normalization in the [AES paper](http://www.aes.org/e-lib/browse.cfm?elib=18609).